packages/gestalt/src/ListText.js

```diff
@@ -0,0 +1,30 @@
+// @flow strict
```

Comment:
This file and the other List subcomponent files should be moved to the `/List` directory

Reply:
OOS for this PR